### PR TITLE
[V3/QoL] Fixes for missing punctuation, spelling, doc string formatting and grammar mistakes.

### PR DESCRIPTION
**Pull request by Redjumpman**
### Type

- [x] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
Multiple fixes for punctuation, spelling docstring formatting,, and grammar mistakes.

### CODE REVIEW

File: redbot/cogs/permissions/permissions.py [hunk 1]
```diff
@@ -254,13 +254,13 @@ def remove_check(self, check_obj: object, before_or_after: str):
...
         if not ctx.message.attachments:
-            return await ctx.send(_("You must upload a file"))
+            return await ctx.send(_("You must upload a file."))
 
         try:
             await yamlset_acl(ctx, config=self.config.owner_models, update=False)
         except Exception as e:
             print(e)
-            return await ctx.send(_("Inalid syntax"))
+            return await ctx.send(_("Inalid syntax."))
```
Comment (tekulvw) [inline redbot/cogs/permissions/permissions.py]: Invalid?

File: redbot/cogs/permissions/permissions.py [hunk 2]
```diff
@@ -280,13 +280,13 @@ def remove_check(self, check_obj: object, before_or_after: str):
...
         if not ctx.message.attachments:
-            return await ctx.send(_("You must upload a file"))
+            return await ctx.send(_("You must upload a file."))
 
         try:
             await yamlset_acl(ctx, config=self.config.guild(ctx.guild).owner_models, update=False)
         except Exception as e:
             print(e)
-            return await ctx.send(_("Inalid syntax"))
+            return await ctx.send(_("Inalid syntax."))
```
Comment (tekulvw) [inline redbot/cogs/permissions/permissions.py]: Invalid?

File: redbot/cogs/permissions/permissions.py [hunk 3]
```diff
@@ -309,13 +309,13 @@ def remove_check(self, check_obj: object, before_or_after: str):
...
         if not ctx.message.attachments:
-            return await ctx.send(_("You must upload a file"))
+            return await ctx.send(_("You must upload a file."))
 
         try:
             await yamlset_acl(ctx, config=self.config.guild(ctx.guild).owner_models, update=True)
         except Exception as e:
             print(e)
-            return await ctx.send(_("Inalid syntax"))
+            return await ctx.send(_("Inalid syntax."))
```
Comment (tekulvw) [inline redbot/cogs/permissions/permissions.py]: Invalid?

File: redbot/cogs/permissions/permissions.py [hunk 4]
```diff
@@ -328,13 +328,13 @@ def remove_check(self, check_obj: object, before_or_after: str):
...
         if not ctx.message.attachments:
-            return await ctx.send(_("You must upload a file"))
+            return await ctx.send(_("You must upload a file."))
 
         try:
             await yamlset_acl(ctx, config=self.config.owner_models, update=True)
         except Exception as e:
             print(e)
-            return await ctx.send(_("Inalid syntax"))
+            return await ctx.send(_("Inalid syntax."))
```
Comment (tekulvw) [inline redbot/cogs/permissions/permissions.py]: Invalid?

File: redbot/cogs/streams/streams.py [hunk 5]
```diff
@@ -394,7 +394,7 @@ def check_name_or_id(self, data: str):
         if current_setting:
             await self.db.role(role).mention.set(False)
             await ctx.send(
-                _("{} will no longer be mentioned " "for a stream alert").format(
+                _("{} will no longer be mentioned " "for a stream alert.").format(
```
Comment (tekulvw) [inline redbot/cogs/streams/streams.py]: Can you remove the `" "`?

**Review by tekulvw — CHANGES_REQUESTED** [review]

File: redbot/cogs/permissions/permissions.py (hunk 1 above)
Comment (Redjumpman) [reply redbot/cogs/permissions/permissions.py]: Done.

File: redbot/cogs/permissions/permissions.py (hunk 2 above)
Comment (Redjumpman) [reply redbot/cogs/permissions/permissions.py]: Done.

File: redbot/cogs/permissions/permissions.py (hunk 3 above)
Comment (Redjumpman) [reply redbot/cogs/permissions/permissions.py]: Done.

File: redbot/cogs/permissions/permissions.py (hunk 4 above)
Comment (Redjumpman) [reply redbot/cogs/permissions/permissions.py]: Done.

File: redbot/cogs/streams/streams.py (hunk 5 above)
Comment (Redjumpman) [reply redbot/cogs/streams/streams.py]: Fixed a bunch of these.

**Comment by Redjumpman** [comment]
@tekulvw Fixed all the stuff you pointed out. Found a bunch of those `" "` in streams. All fixed.

**Comment by Redjumpman** [comment]
having trouble rebasing. Closing this to reopen in a bit.

**Pull request closed without merging**